### PR TITLE
♻️ Pass `storage_root` to `access_aws`

### PR DESCRIPTION
**Pull request by falexwolf**

### CODE REVIEW

**Review by fredericenard — APPROVED** [review]

File: lamindb_setup/dev/_hub_core.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def _load_instance(
     return instance, storage
 
 
-def access_aws(access_token: Optional[str] = None) -> Dict[str, str]:
+def access_aws(storage_root: str, access_token: Optional[str] = None) -> Dict[str, str]:
     from .._settings import settings
 
     if settings.user.handle != "anonymous" or access_token is not None:
```
Comment (Koncopd) [inline lamindb_setup/dev/_hub_core.py:214]: `storage_root` is not passed below, but actually should, i think.

Comment (Koncopd) [reply lamindb_setup/dev/_hub_core.py:214]: `credentials = call_with_fallback_auth(_access_aws, storage_root=storge_root, access_token=access_token)`

Comment (falexwolf) [reply lamindb_setup/dev/_hub_core.py:214]: Right!

Comment (falexwolf) [reply lamindb_setup/dev/_hub_core.py:214]: It got removed resolving conflicts with main.

**Pull request merged**